### PR TITLE
Normalizing optimized cone direction

### PR DESCRIPTION
**Pull request by nenadpol**
Normalizing optimized cone coefficients in void optimizeModelCoefficients. Proposed code starts line nr. 339 and end line nr 343

### CODE REVIEW

File: sample_consensus/include/pcl/sample_consensus/impl/sac_model_cone.hpp [hunk 1]
```diff
@@ -335,6 +335,13 @@ pcl::SampleConsensusModelCone<PointT, PointNT>::optimizeModelCoefficients (
   PCL_DEBUG ("[pcl::SampleConsensusModelCone::optimizeModelCoefficients] LM solver finished with exit code %i, having a residual norm of %g. \nInitial solution: %g %g %g %g %g %g %g \nFinal solution: %g %g %g %g %g %g %g\n",
              info, lm.fvec.norm (), model_coefficients[0], model_coefficients[1], model_coefficients[2], model_coefficients[3],
              model_coefficients[4], model_coefficients[5], model_coefficients[6], optimized_coefficients[0], optimized_coefficients[1], optimized_coefficients[2], optimized_coefficients[3], optimized_coefficients[4], optimized_coefficients[5], optimized_coefficients[6]);
+
+    Eigen::Vector3f line_dir (optimized_coefficients[3], optimized_coefficients[4], optimized_coefficients[5]);
```
Comment (taketwo) [inline sample_consensus/include/pcl/sample_consensus/impl/sac_model_cone.hpp]: Please fix indentation in this line and remove extra blank line below. Thanks!

**Comment by jspricke** [comment]
looking good, can you please squash your commits into one?

**Comment by nenadpol** [comment]
ok, I'll do that

cheers 
Nenad

On Thursday, August 27, 2015 10:31 CEST, Jochen Sprickerhof <notifications@github.com> wrote:
 
looking good, can you please squash your commits into one?

**Comment by nenadpol** [comment]
Hi Jochen,

GitHub is unknown to me.
How do I squash commits into one?

cheers
Nenad

On Thursday, August 27, 2015 10:31 CEST, Jochen Sprickerhof <notifications@github.com> wrote:
 
looking good, can you please squash your commits into one?

**Comment by jspricke** [comment]
I've done it for you. You can read up on git squash (it's a git feature, not specific to Github) for example here: http://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Pull request closed without merging**